### PR TITLE
Editing docs

### PR DESCRIPTION
**Pull request by zaelgohary**
### Description

Editing Dashboard docs

### Changes

Adding docs for contribution, workflows, releasing, and editing the getting_started

### Related Issues

- https://github.com/threefoldtech/tfgrid_dashboard/issues/449

### Checklist

- [x] Tests included
- [x] Build pass
- [x] Documentation
- [x] Code format and docstrings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,61 @@
-## THREEFOLD GRID DASHBOARD
+# TFGrid_Dashboard
```
Comment (xmonader) [inline README.md]: let's remove the underscore

File: README.md [hunk 2]
```diff
@@ -1,76 +1,61 @@
...
-- Transfer TFT within the TFChain
-- Swap TFT to and from Cosmos and Stellar
-- Create/delete/edit/view/search twins, farms, farm public IPs, farm nodes and farm nodes public configuration.
-- Add Stellar payout to farms
-- Reserve/Unreserve dedicated nodes.
-- Explore the grid capacity (i.e view the TF Grid statistics, nodes and farms across the world).
+The Dashboard is a unified admin interface for everything related to Threefold Grid such as Portal, Explorer, Calculators, Zero-OS Bootstrap & Playground. For more details, check [Dashboard Manual](https://library.threefold.me/info/manual/#/manual__dashboard_readme)
 
-If you don't know where to start, you can also review the TF Chain manual before even connecting to polkadot.js.
+- **Clone the repository**
```
Comment (xmonader) [inline README.md]: I think this section is misplaced

File: docs/releasing.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Releasing process
```
Comment (xmonader) [inline docs/releasing.md]: Releasing -> Release

File: README.md (hunk 2 above)
Comment (zaelgohary) [reply README.md]: removed from README as it's already included in getting_started doc

**Review by xmonader — APPROVED** [review]

**Pull request merged**